README.md

```diff
@@ -1,7 +1,38 @@
-Looking to embed Tabzilla in your website? Check out the documentation at http://bedrock.readthedocs.org/en/latest/tabzilla.html.
+# Tabzilla
 
----
+This is the home of Tabzilla, the universal tab you can add to your website. Below
+follows simple instructions on how to get, and use Tabzilla.
 
-Tabzilla is no longer maintained in this separate repo. It is now integrated into the Bedrock repo:
+## Bower Package
 
-http://github.com/mozilla/bedrock/
+Do you use [Bower](http://bower.io/) to manage your front-end dependencies? Well, then you will be
+happy to hear that you can install Tabzilla using Bower.
+
+```Shell
+bower install mozilla-tabzilla
+```
+
+That's it! Now you have all the bits you need. Next step, add it to your site.
+
+## Manual Download
+
+Don't use [Bower](http://bower.io/)? No problem, you can [download the latest release right here from the releases page](https://github.com/schalkneethling/tabzilla/releases).
```

Comment:
Yes, those will need to be changed but, I am thinking perhaps a small follow on, once all of this has been merged?